### DISCUSSION TITLE
文章的排版样式

### DISCUSSION THREAD

**Discussion by zyyaria** [hugo-next/hugo-theme-next#52] [Q&A]
- 请问文章的排版样式在哪里可以修改？
- 超链接后面的那个箭头可以取消吗？
- 支不支持其他的markdown主题呢？要怎么替换呢？

**Comment by elkan1788** [comment]
关于1，3问题能否说的具体一些，可以截图说明一下。
链接后面的箭头可以自定义配置的，参配置里的说明操作即可：
![image](https://user-images.githubusercontent.com/1217096/187556278-05f905cc-6228-4285-b727-b156b96163be.png)
